docs/build/custom.md

```diff
@@ -113,80 +113,122 @@ The `MinSizeRel` configuration will produce the smallest binary size.
 
-```bash
-pip install -U .\build\Windows\MinSizeRel\MinSizeRel\dist\onnxruntime-1.7.0-cp37-cp37m-win_amd64.whl
-```
+    The build options are specified with the file provided to the `--build_settings` option. See the options used by the pre-built mobile package at [tools/ci_build/github/android/default_mobile_aar_build_settings.json](https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/android/default_mobile_aar_build_settings.json). You can use this file directly.
 
-## Version of ONNX Runtime to build from
+2. Use the local custom Android AAR package.
 
-Unless there is a specific feature you need, do not use the unreleased `master` branch.
+    For example, in an Android Studio project:
 
-Once you have cloned the ONNX Runtime repo, checkout one of the release tags to build from.
+    a. Copy the AAR file from `/path/to/working/dir/output/aar_out/<build config, e.g., Release>/com/microsoft/onnxruntime/onnxruntime-mobile/1.11.0/onnxruntime-mobile-1.11.0.aar` to the project's `<module name, e.g., app>/libs` directory.
 
-```bash
-git clone --recursive https://github.com/microsoft/onnxruntime
-cd onnxruntime
-git checkout <release tag>
-```
+    b. Update the project's `<module name>/build.gradle` file dependencies section:
 
-Release tag names follow the pattern `v<release version>`. For example, `v1.11.0`.
-Find them [here](https://github.com/microsoft/onnxruntime/tags).
+    ```diff
+    -    implementation 'com.microsoft.onnxruntime:onnxruntime-mobile:latest.release'
+    +    implementation files('libs/onnxruntime-mobile-1.11.0.aar')
+    ```
 
-## Example build commands
+### Python
 
-### Build on windows, with reduced operator support, and support for ORT format models only
+If you wish to use the ONNX Runtime python bindings with a minimal build, exceptions must be enabled due to Python requiring them.
 
-`<ONNX Runtime repository root>.\build.bat --config=MinSizeRel --cmake_generator="Visual Studio 16 2019" --build_shared_lib --minimal_build --disable_ml_ops --disable_exceptions --include_ops_by_config <config file from model conversion> --skip_tests`
+Remove `--disable_exceptions` and add `--build_wheel` to the build command in order to build a Python Wheel with the ONNX Runtime bindings.
 
-### Linux
+A .whl file will be produced in the build output directory under the `<config>/dist` folder.
 
-`<ONNX Runtime repository root>./build.sh --config=MinSizeRel --build_shared_lib --minimal_build --disable_ml_ops --disable_exceptions --include_ops_by_config <config file from model conversion> --skip_tests`
+* The Python Wheel for a Windows MinSizeRel build using build.bat would be in `<ONNX Runtime repository root>\build\Windows\MinSizeRel\MinSizeRel\dist\`
+* The Python Wheel for a Linux MinSizeRel build using build.sh would be in `<ONNX Runtime repository root>/build/Linux/MinSizeRel/dist/`
```

Comment:
nit: not sure whether we should default to MinSizeRel or Release in the doco. if you really really need the smallest possible it's MinSizeRel, but that potentially comes with a perf cost and Release might be better in general.